src/dos/cdrom_ioctl_linux.cpp

```diff
@@ -113,4 +291,154 @@ bool CDROM_Interface_Ioctl::ReadSectorsHost([[maybe_unused]] void *buffer,
 	return false; /*TODO*/
 }
 
+void CDROM_Interface_Ioctl::InitAudio()
+{
+	// Find a a channel name that doesn't exist (channel names must be
+	// unique or they get overwritten)
+	mixer_channel_t exists;
+	int num                  = 0;
+	const std::string prefix = MixerChannelNamePrefix;
```

Comment:
Why not just use `MixerChannelNamePrefix` directly below?

Reply:
Because it needs to be converted to `std::string`

Reply:
Ah ok.